### PR TITLE
Logging refactor

### PR DESCRIPTION
**Pull request by akallabeth**

### CODE REVIEW

File: libfreerdp/codec/dsp_ffmpeg.c [hunk 1]
```diff
@@ -664,8 +664,7 @@ static BOOL freerdp_dsp_channel_mix(FREERDP_DSP_CONTEXT* context, const BYTE* sr
 
 			case 2:  /* We only support stereo, so we can not handle this case. */
 			default: /* Unsupported number of channels */
-				WLog_WARN(TAG, "[%s] unsuported source channel count %" PRIu16, __FUNCTION__,
-				          srcFormat->nChannels);
+				WLog_WARN(TAG, "unsuported source channel count %" PRIu16, srcFormat->nChannels);
```
Comment (hardening) [inline libfreerdp/codec/dsp_ffmpeg.c:667]: s/unsuported/unsupported/

File: libfreerdp/codec/dsp_ffmpeg.c [hunk 2]
```diff
@@ -693,8 +692,7 @@ static BOOL freerdp_dsp_channel_mix(FREERDP_DSP_CONTEXT* context, const BYTE* sr
 
 		case 1:  /* Invalid, do we want to use a 0 channel sound? */
 		default: /* Unsupported number of channels */
-			WLog_WARN(TAG, "[%s] unsuported channel count %" PRIu16, __FUNCTION__,
-			          srcFormat->nChannels);
+			WLog_WARN(TAG, "unsuported channel count %" PRIu16, srcFormat->nChannels);
```
Comment (hardening) [inline libfreerdp/codec/dsp_ffmpeg.c:695]: s/unsuported/unsupported/

File: libfreerdp/core/certificate.c [hunk 3]
```diff
@@ -541,7 +541,7 @@ static BOOL certificate_write_server_public_signature(const rdpCertificate* cert
 	Stream_Write_UINT16(s, wSignatureBlobType);
 	Stream_Write_UINT16(s, wSignatureBlobLen);
 
-	WLog_WARN(TAG, "[%s] TODO: Calcualte proper signature", __FUNCTION__);
+	WLog_WARN(TAG, "TODO: Calcualte proper signature");
```
Comment (hardening) [inline libfreerdp/core/certificate.c:545]: s/Calcualte/Compute/

**Review by hardening — APPROVED** [review]

File: libfreerdp/core/certificate.c (hunk 3 above)
Comment (akallabeth) [reply libfreerdp/core/certificate.c:545]: fixed

File: libfreerdp/codec/dsp_ffmpeg.c (hunk 2 above)
Comment (akallabeth) [reply libfreerdp/codec/dsp_ffmpeg.c:695]: fixed

**Pull request merged**